app/views/SidebarView/index.js

```diff
@@ -103,9 +105,16 @@ class Sidebar extends Component {
 		if (nextState.isAdmin !== isAdmin) {
 			return true;
 		}
+		if (nextProps.state !== state && isSetAdminNeedCall === true) {
```

Comment:
As you can see at the beginning of the block, `state` comes from `react-navigation` (it's track current state of the navigation) and it's an object.
`nextProps.state !== state` is an object comparison, so it'll never be equal.
![image](https://user-images.githubusercontent.com/804994/107281273-35e75d00-6a38-11eb-9bd2-5d9885a32a68.png)